components/CreateEventForm.tsx

```diff
@@ -63,7 +63,7 @@ const CreateEventForm: React.FC<CreateEventFormProps> = ({ uid }) => {
     const eventCollectionRef = collection(userDocRef, 'events');
     const CreatedEventRef = await addDoc(eventCollectionRef, {
       event_name: eventName,
-      timestamp: datetime.toLocaleString()
+      timestamp: datetime.toLocaleString('en-GB')
```

Comment:
wait so in the version before with no argument, the date will be formatted according to the user's locale settings right. Just to double check, we're changing this to en-GB so that there will be no mismatch cases with the date sorting function for users in countries with the MM/DD/YYYY format, correct me if I'm wrong please :0

Reply:
yes, so now eveyrone's date will be stored as `DD/MM/YYYY, ....`, I'll be highlighting this in our README as this is quite the software engineering practice we are following as we are thinking about backward compatibility in our code instead of wiping the database.